src/components/Button/Button.scss

```diff
@@ -0,0 +1,47 @@
+@import '../../variables.module.scss';
+
+.copernicus-button {
+  padding: 1px 2px;
+  height: 35px;
+  min-width: 109px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+
+  &.primary {
+    background-color: $whiteBackground;
+    color: $primaryColor;
+    border: 1px solid $primaryColor;
+
+    &:hover {
+      color: $whiteColor;
+      background-color: $primaryColor;
+    }
+  }
+
+  &.danger {
+    background-color: $errorCol;
+    color: $textColor;
```

Comment:
I think it would be better to have this as $whiteColor and on button hover it would go a bit darker (similar to success)